Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1204,7 +1204,7 @@ final class LayerTests: XCTestCase {
           biasInitializer: zeros())
         )
         withTensorLeakChecking {
-            let (outputs, _) = rnn.valueWithPullback(at: inputs) { rnn, inputs in
+            let (outputs, _) = valueWithPullback(at: rnn, inputs) { rnn, inputs in
```

Comment:
@dan-zheng you missed a `valueWithPullback` on line 1176.

Reply:
Thanks for the catch! Done in https://github.com/tensorflow/swift-apis/pull/577/commits/0e75b1b91296c823238640a6c51f2a5129b23054.